src/grammar.lalrpop

```diff
@@ -499,9 +499,17 @@ ClauseGuard4: UntypedClauseGuard = {
         right: Box::new(right),
     },
 
+    <name:ClauseGuard5> <s:@L> "." <index:PositiveRawInt> <e:@L> => ClauseGuard::TupleIndex {
+        location: location(s, e),
+        index,
+        typ: (),
+        tuple: Box::new(name),
+    },
+
     ClauseGuard5 => <>,
 }
 
+#[inline]
```

Comment:
This is the first time I'm learning about `inline`. Do you know where I can find the documentation for it? I always struggle with finding info about lalrpop.

Reply:
i found this blog http://smallcultfollowing.com/babysteps/blog/2016/03/02/nice-errors-in-lalrpop/

Reply:
Ah nice! Thank you